### PR TITLE
Change static files serving to whitenoise plugin

### PR DESCRIPTION
**Pull request by korgan00**
### Summary

Right now we have some limitations in the way we are serving static files because Django doesn't serve them by default. I use a plugin to implement better static files sharing.

### Details and comments

Using [whitenoise for django](https://whitenoise.readthedocs.io/en/stable/django.html) plugin to manage static files.

### CODE REVIEW

File: gateway/main/urls.py [hunk 1]
```diff
@@ -49,7 +49,6 @@
     path("", include("django_prometheus.urls")),
     path("backoffice/", admin.site.urls),
     re_path(r"^api/v1/", include(("api.v1.urls", "api"), namespace="v1")),
-    re_path(r"^static/(?P<path>.*)$", serve, {"document_root": settings.STATIC_ROOT}),
```
Comment (Tansito) [inline gateway/main/urls.py:52]: is this not need it anymore, Goyo?

**Comment by psschwei** [comment]
Will this help with the issues in #1236 ?

**Comment by korgan00** [comment]
I cannot be sure, but my intuition say that it should. Right now it works with docker.

**Comment by psschwei** [comment]
Could we also check against Kubernetes? We should also test there since that's where we ultimately deploy. Docker can be useful for running code locally, but otherwise it can also get you into a bit of trouble down the line too...

You can use Kind to run a cluster locally.

**Comment by psschwei** [comment]
ok, just checked and the docs DO show up on kubernetes :tada: 

image

**Review by psschwei — APPROVED** [review]

**Comment by Tansito** [comment]
@psschwei could you double check too that the panel administrator loads correctly? 🙏

**Comment by psschwei** [comment]
@Tansito that would be `/backoffice` right? (so locally, `http://localhost:8000/backoffice`)

**Comment by Tansito** [comment]
Yes!

**Comment by psschwei** [comment]
admin panel works

image

**Comment by Tansito** [comment]
It worked with `DEBUG=0`, @psschwei ? Sorry for the double check, I remember having problems with that debug configuration.

**Comment by psschwei** [comment]
No -- for my local setup I default to DEBUG=1, so still need to check with DEBUG=0...

**Review by psschwei — APPROVED** [review]
Confirmed that admin panel works with DEBUG=0

image

**Review by Tansito — APPROVED** [review]
Awesome! Thank you @psschwei for the double check, really appreciate it. And congrats @korgan00 for your first PR 🎉

**Pull request merged**